projects/packages/forms/src/dashboard/about/style.scss

```diff
@@ -186,9 +186,14 @@
 						height: 100%;
 					}
 
-					.akismet-icon {
+					.akismet-icon,
+					.creative-mail-icon {
 						clip-path: circle(16px at center);
 					}
+
+					.jetpack-icon {
+						fill: var(--jp-green-40);
+					}
```

Comment:
Wondering how often this CSS would be needed for the icons, and if often, whether any of this should be built into the svgs. Could be a separate PR if so.

Reply:
The green fill was only necessary on the dashboard, not sure why :thinking: